@xen-orchestra/lite/src/libs/xen-api/xen-api.types.ts

```diff
@@ -112,6 +112,13 @@ export interface XenApiPool extends XenApiRecord<'pool'> {
   name_label: string
   other_config: Record<string, string>
   default_SR: XenApiSr['$ref']
+  tags: Array<string>
+  name_description: string
+  ha_enabled: string
+  migration_compression?: boolean
+  suspendSr: XenApiSr['$ref']
+  crash_dump_SR: XenApiSr['$ref']
```

Comment:
The `suspendSr` in XO 6 relates to `suspend_image_SR` in XO Lite:
```suggestion
  suspend_image_SR: XenApiSr['$ref'] | OPAQUE_REF_NULL
  crash_dump_SR: XenApiSr['$ref'] | OPAQUE_REF_NULL
```